### PR TITLE
feat: Generate release redirects at build time with latest release from Github

### PR DESCRIPTION
**Pull request by jbleuzen**
I've noticed that you update manually links to the latest release with version.
This PR will use the latest release version directly from Github api.

One thing that I was not able to retrieve is the md5 checksum. 
How do you calculate it, from what I know there is no way to retrieve it from API (but I may be wrong) ?

### CODE REVIEW

File: _data/github.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const fetch = require("node-fetch");
+
+module.exports = async function() {
+  console.log( "Fetching new github stargazers count…" );
```
Comment (j0k3r) [inline _data/github.js:4]: Wrong comment

File: _includes/_download.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
       <div class="col-md-12 text-center wp4">
         <h1>Want to give it a try?</h1>
         <p><a href="https://www.wallabag.it/" class="download-btn">REGISTER <i class="fa fa-user-plus"></i></a> <a href="https://doc.wallabag.org/en/admin/installation/installation.html#installation" class="download-btn">DOWNLOAD <i class="fa fa-download"></i></a><br></p>
-        <p>Last release: wallabag 2.5.4<br>
+        <p>Last release: wallabag {{ github.appVersion }}<br>
           (md5 checksum: 5e159c59822c92154b200eb46f596c27)<br>
```
Comment (j0k3r) [inline _includes/_download.html:8]: Problem is we can't determine this one unless we download the package from the release and calculate the md5. But maybe it's doable. But the deploy process will be slower.

Comment (jbleuzen) [reply _includes/_download.html:8]: Fixed, thanks !

File: _data/github.js (hunk 1 above)
Comment (jbleuzen) [reply _data/github.js]: Fixed thanks

File: _includes/_download.html (hunk 2 above)
Comment (jbleuzen) [reply _includes/_download.html:8]: Ok, I'll see what I can come up with in another PR

**Review by j0k3r — APPROVED** [review]

**Comment by nicosomb** [comment]
@jbleuzen you have conflicts because we released 2.6.1

**Comment by nicosomb** [comment]
Thank you @jbleuzen !

**Review by j0k3r — APPROVED** [review]

**Pull request merged**